cms/static/js/views/content_group_details.js

```diff
@@ -75,18 +75,18 @@ define([
         },
 
         getOutlineAnchorMessage: function () {
-            var message = gettext(
+            var message = _.escape(gettext(
                     /*
                         Translators: 'outlineAnchor' is an anchor pointing to
                         the course outline page.
                     */
                     'This content group is not in use. Add a content group to any unit from the %(outlineAnchor)s.'
-                ),
+                )),
                 anchor = str.sprintf(
                     '<a href="%(url)s" title="%(text)s">%(text)s</a>',
                     {
                             url: this.model.collection.parents[0].outlineUrl,
-                            text: gettext('Course Outline')
+                            text: _.escape(gettext('Course Outline'))
```

Comment:
@cahrens FYI: When you do the additional work during the marathon, this is a great candidate for Andy's new `HtmlUtils.interpolateHtml()`.  Hopefully we'll get that landed and documented in time.